windows-msi/msi.wxs

```diff
@@ -62,6 +62,11 @@
         <Property Id="DISABLEROLLBACK" Value="yes"/>
         <Property Id="EDITOR" Value="notepad.exe"/>
         <PropertyRef Id="NETFRAMEWORK40FULL"/>
+        <?if $(sys.BUILDARCH)~="x86" Or $(sys.BUILDARCH)~="x64" ?>
```

Comment:
Maybe we can drop binary checksum advertising completely? Some linkers set it, some don't. IIRC, files without checksum are always copied on repair. Savings are miniscule.

Reply:
Agreed.